templates/apache-guacamole/0/README.md

```diff
@@ -0,0 +1,21 @@
+# Apache Guacamole
+
+Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.
+We call it clientless because no plugins or client software are required.
+Thanks to HTML5, once Guacamole is installed on a server, all you need to access your desktops is a web browser.
+
+This service uses the official Apache [http://guacamole.incubator.apache.org](https://hub.docker.com/r/glyptodon/guacamole/) Guacamole image. It also uses the official MariaDB image as its backend.
+Health checks are enabled on all services.
+## Prerequisites
+You must have the Rancher-NFS storage driver set up. This is required for persistent SQL storage, otherwise all users and connection settings will be reset on every container move.
```

Comment:
Can you update the README if the template is no longer dependent on rancher-nfs, thanks!